src/test/java/com/wikia/webdriver/elements/mercury/old/PortableInfoboxObject.java

```diff
@@ -235,6 +235,9 @@ public PortableInfoboxObject isVideoCaptionVisible() {
     return this;
   }
 
+  /**
+   * Hero image is the image that appears on the top of infobox
+   */
   public PortableInfoboxObject isHeroImageCentered() {
     Assertion.assertEquals(images.get(0).getCssValue("vertical-align"), "middle");
     PageObjectLogging.log("Hero image", "is centered", true);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Hero image" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)